### PR TITLE
docs(api): Document updates to assetEmitted hook

### PR DESCRIPTION
**Pull request by superburrito**
PR for https://github.com/webpack/webpack.js.org/issues/3278.

Am aware that this should be merged into `next` instead -- seems like `master` already has written docs for `assetEmitted` which haven't been merged into `next` (https://github.com/webpack/webpack.js.org/blob/next/src/content/api/compiler-hooks.md).

@EugeneHlushko I figure you'd want to update `next` with your changes first?

### CODE REVIEW

**Comment by EugeneHlushko** [comment]
Yeap we update next time to time, let me handle that tonight

**Comment by superburrito** [comment]
Whenever's good @EugeneHlushko

File: src/content/api/compiler-hooks.md [hunk 1]
```diff
@@ -252,9 +253,21 @@ Called after emitting assets to output directory.
 
 `AsyncSeriesHook`
 
-Allows to get byte content of emitted asset. Available since webpack v4.39.0
+Executed when an asset has been emitted. Provides access to information about the emitted asset, such as its output path and byte content. 
+
+- Callback Parameters: `file`, `info`
```
Comment (montogeek) [inline src/content/api/compiler-hooks.md]: Would be good to add more information to `info`.

**Comment by montogeek** [comment]
@superburrito Friendly ping

**Comment by EugeneHlushko** [comment]
Thanks, we will iterate on it if needed

**Pull request merged**